packages/block-library/src/media-text/edit.js

```diff
@@ -318,6 +365,17 @@ function MediaTextEdit( { attributes, isSelected, setAttributes } ) {
 
 	const blockEditingMode = useBlockEditingMode();
 
+	function resetAll() {
+		setAttributes( {
+			isStackedOnMobile: false,
+			imageFill: false,
+			mediaAlt: undefined,
+			focalPoint: undefined,
+			mediaWidth: undefined,
```

Comment:
`mediaWidth` must be explicitly given a default value of 50. Otherwise, the block will be broken when resetting all.


https://github.com/WordPress/gutenberg/assets/54422211/4f9ce904-54b3-4286-9729-4ff9cc415c44